src/components/text-editor/test-setup/editor-view-builder.ts

```
@@ -0,0 +1,143 @@
+ * This should be called in test cleanup/afterEach.
+ *
+ * @param view - The editor view to destroy
+ * @param container - The container element to remove (if created by test)
+ */
+export function cleanupEditorView(
+    view: EditorView,
+    container?: HTMLElement,
+): void {
+    view.destroy();
+
+    if (container && container.parentNode) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)